File: 1.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        if len(nums) == 2 and sum(nums) == target:
            return [0,1]
        res = list()
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    res = [i,j]
        return res

File: 1.go
func twoSum(nums []int, target int) []int {
    if len(nums) == 2 && nums[0]+nums[1] == target {
        return []int{0, 1}
    }
    res := []int{}
    for i := 0; i < len(nums); i++ {
        for j := i + 1; j < len(nums); j++ {
            if nums[i]+nums[j] == target {
				return []int{i,j}
            }
        }
    }
    return res
}
